api/src/main/java/org/opennms/integration/api/v1/flows/Flow.java

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.integration.api.v1.flows;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+
+import org.opennms.integration.api.v1.annotations.Consumable;
+
+/**
+ * A network flow.
+ *
+ * Represents statistics about the communication between two systems.
+ *
+ * @since 1.0.0
+ */
+@Consumable
```

Comment:
Can be annotated with `@Model` instead

Reply:
Can you explain some more? My understanding is, this interface is always implemented by core and never meant to be instantiated by the plugin.

Reply:
`Consumable` was normally used for services to be pulled from the registry i.e. https://github.com/OpenNMS/opennms-integration-api/blob/master/api/src/main/java/org/opennms/integration/api/v1/distributed/KeyValueStore.java#L39

`Model` is used to identify object model type interfaces i.e. https://github.com/OpenNMS/opennms-integration-api/blob/master/api/src/main/java/org/opennms/integration/api/v1/model/Node.java#L45

Reply:
Got it. Thanks.

Does this imply that there should be a implementation in `commons`? Currently my there is no need to ever instantiate a flow in a plugin.

Reply:
`commons` implementation not necessary